features/partnerhistorie/peildatum-periode-filtering.feature

```diff
@@ -0,0 +1,258 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Selecteer het/de correcte (historische) huwelijken geregistreerde partnerschappen.
```

Comment:
Bedoel je: Selecteer (historische) huwelijken/ geregistreerde partnerschappen.

Reply:
Ja, ik weet eigenlijk niet meer waarom ik het woordje 'correcte' hier gebruik.
Is aangepast.